src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -604,6 +604,15 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 					return m_polyMesh.subdivAdaptiveError == 0.0f || m_polyMesh.subdivAdaptiveSpace == g_objectSpace;
 				}
 			}
+			else if( const IECore::ExternalProcedural *procedural = IECore::runTimeCast<const IECore::ExternalProcedural>( object ) )
+			{
+				// We don't instance "ass archive" procedurals, because Arnold
+				// does automatic instancing of those itself, using its procedural
+				// cache.
+				return
```

Comment:
Not only does it fix that, but it turns out it fixes another outstanding issue with IPR and lights inside ass archives.